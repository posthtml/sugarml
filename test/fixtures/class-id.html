#foo.bar(class='foo1' id='bar2')
