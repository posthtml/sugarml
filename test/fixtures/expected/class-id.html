<div id="foo bar2" class="bar foo1"></div>
